feature-libs/organization/administration/components/shared/sub-list/assign-cell.component.ts

```diff
@@ -88,11 +88,14 @@ export class AssignCellComponent<T> extends CellComponent {
    * item.
    */
   protected get link(): string {
-    return (
-      this.outlet.context.code ??
-      this.outlet.context.customerId ??
-      this.outlet.context.uid
-    );
+    let contextData: TableDataOutletContext | undefined;
+    this.outlet.context$
+      .subscribe((context) => (contextData = context))
+      .unsubscribe();
+    if (contextData) {
+      return contextData.code ?? contextData.customerId ?? contextData.uid;
+    }
+    return '';
   }
```

Comment:
This method is synchronous,  but the context$ might be potentially asynchronous.
So I’d rather expose link$ observable and pipe async to it in the template.
btw. context$ is emitting on any the change of the object, so you’d need to distinctUntilChanged only on the property you’re picking.
This change from static methods to observables is obviously a breaking change, requireing migrations, and docs.
And it’s also a refactor per se (affecting the timing of change detection), which require regression testing in all places that use this new observable instead of static property.